ingest/validation/validate_metadata.py

```diff
@@ -332,6 +364,12 @@ def cast_metadata_type(metadatum, value, id_for_error_detail, convention, metada
             # files that support array-based metadata navtively (eg. loom,
             # anndata etc) splitting on pipe may become problematic
             for element in value.split('|'):
+                try:
+                    if 'ontology' in convention['properties'][metadatum]:
+                        element = regularize_ontologyID(element)
+                except KeyError:
+                    # non-metadata convention metadata will trigger this exception
```

Comment:
"Non-metadata-convention" is somewhat wordy.  How about this?

```suggestion
                    # unconventional metadata will trigger this exception
```

While "unconventional" by itself is unclear, but "unconventional metadata" seems fairly unambiguous.

If you agree, a global find/replace would make this change consistently.

Reply:
addressed in 639a006